gmaps/directions.py

```diff
@@ -88,17 +101,29 @@ def _handle_layer_status(self, change):
                 "No directions returned: " + change["new"])
 
 
-def _directions_options(start, end, waypoints):
+def _directions_options(start, end, waypoints, avoid_ferries=False,
+                        avoid_highways=False, avoid_tolls=False,
+                        optimize_waypoints=False):
```

Comment:
Since this only gets called internally and we've explicitly bubbled the defaults up to the `directions_layer` function, we don't need the default values in the type signature here: `_directions_options` always gets called with all the arguments explicitly. 

I think it therefore makes sense to remove the defaults?

```python
def _directions_options(start, end, waypoints, avoid_ferries, avoid_highways, avoid_tolls, optimize_waypoints)
```